gql/queries/FTVAEventList.gql

```diff
@@ -1,6 +1,10 @@
 # import "../gql/fragments/Image.gql"
 
 query FTVAEventList {
+  entry(section: ["ftvaListingEvents"]) {
+    titleGeneral
+    summary
+  }
   entries(section: ["ftvaEvent"], orderBy: "postDate") {
```

Comment:
we need to either comment or remove entries(section: ["ftvaEvent"], orderBy: "postDate") query portion